react/components/Prefetch/PrefetchContext.tsx

```diff
@@ -50,17 +48,13 @@ const HALF_HOUR_MS = 1000 * 60 * 30
 const state: PrefetchState = {
   routesCache: new LRUCache({ max: 100, maxAge: HALF_HOUR_MS }),
   pathsCache: {
-    product: new LRUCache({
-      max: 100,
-      dispose: disposeFn,
-      maxAge: HALF_HOUR_MS,
-    }),
-    search: new LRUCache({ max: 75, dispose: disposeFn, maxAge: HALF_HOUR_MS }),
-    other: new LRUCache({ max: 75, dispose: disposeFn, maxAge: HALF_HOUR_MS }),
+    product: new LRUCache({ max: 100, disposeFn, maxAge: HALF_HOUR_MS }),
+    search: new LRUCache({ max: 75, disposeFn, maxAge: HALF_HOUR_MS }),
+    other: new LRUCache({ max: 75, disposeFn, maxAge: HALF_HOUR_MS }),
   },
   pathsState: {},
   routePromise: {},
-  queue: new PQueue({ concurrency: MAX_CONCURRENCY, autoStart: false }),
+  queue: new PQueue(),
```

Comment:
Is this `PQueue` something WIP?

Reply:
yes, we could not solve, this previous lib we used was not meant to be used in a browser and did not have a valid es5 implementation. Someone that wants to put this in production, will need to solve this on its own.